src/prefect/cli/task.py

```diff
@@ -0,0 +1,53 @@
+from typing import List
+
+import typer
+
+from prefect.cli._types import PrefectTyper
+from prefect.cli.root import app
+from prefect.task_server import serve as task_serve
+from prefect.utilities.importtools import import_object
+
+task_app = PrefectTyper(name="task", help="Commands for working with task scheduling.")
+app.add_typer(task_app, aliases=["task"])
+
+
+@task_app.command()
+async def serve(
+    entrypoints: List[str] = typer.Argument(
+        ...,
+        help="The paths to one or more tasks, in the form of `./path/to/file.py:task_func_name`.",
+    ),
+):
+    """
+    Starts a task server that runs scheduled tasks.
+
+    This command initiates a task server which will monitor and execute the specified tasks.
+
+    Args:
+        entrypoints: List of strings representing the paths to one or more
```

Comment:
It would be cool if you could give it a file and it would serve any tasks in there, but we can take a shot at that later.

Reply:
I started down that rabbit hole and then I had the same though, best to have something simple to start.